config/index.html

```diff
@@ -68,4 +67,5 @@ <h2 class="container--title">Configurações</h2>
     <p>Feito com ❤ por <a href="https://links.ernane.dev/" target="_blank">Ernane Ferreira</a></p>
   </footer>
 </body>
+<script src="../assets/scripts/config.js"></script>
```

Comment:
With the previously suggested change, we would not need this repositioning of the settings script insertion at the end of the html. It would be loaded normally, but its execution would only be performed at the end of the complete page load. Loading events, for this type of project, need to be well organized to avoid future confusion.